src/lib/middleware/assetMiddleware.ts

```diff
@@ -4,7 +4,7 @@ import chalk from "chalk"
 
 const { CDN_URL, NODE_ENV } = process.env
 
-export function assetMiddleware(manifestFileName = "manifest.json") {
+export default function assetMiddleware(manifestFileName = "manifest.json") {
```

Comment:
Why `default`?

Reply:
I plan of normalizing all the current middlewares to follow the same pattern at which point a long term decision can be made. I would be hesitant to avoid language features because they are "difficult" to use and instead suggest we rely of tools such as linters and IDEs.

Reply:
(blocking) For all new files being created we use named exports, per long-term artsy convention. As we discussed in slack, I'd encourage you to open an RFC for changing this pattern.

Reply:
`export default` is scattered throughout the force code base and it used quite heavily in our newer V2 code, which due to the [bunny theory of code](https://humanwhocodes.com/blog/2015/05/14/the-bunny-theory-of-code/) will be duplicated over time.  

In this case, the majority of our middleware use `module.exports` and keeping with consistency using `export defaults` for the few new ones makes sense in this case. If we wish to do away with `export default` let us make a concerted effort to clean up our codebase once and for all.